docs/index.md

```diff
@@ -223,7 +223,7 @@ Meta filtering also automatically follows struct pointers, so indirect access to
 structures pointed by an `sk_buff` field is possible.
 
 ```none
-$ retis collect -m 'sk_buff.dev.nd_net.net.ns.inum == 4026531840'
+$ retis collect -m 'sk_buff.dev.nd_net.net.ns.inum == 0xF0000000'
```

Comment:
Not sure about this one. While showing hex can be used is OK, netns ids are rarely used in the hex form IMO.

Reply:
IDK, it's an inode number between:
`PROC_DYNAMIC_FIRST and 0xffffffff`

Multiple places refer to it in hex:
https://github.com/torvalds/linux/blob/90d35da658da8cff0d4ecbb5113f5fac9d00eb72/fs/nfs/callback.c#L176

Not sure this is bad. At least the upside is that shows how hexes are required in exprs.

Reply:
anyways, no strong opinions about this. We can always add a separate line for it.

Reply:
It's just that I always saw netns being referenced as a decimal value (eg. `lsns`) and as a user I find using an hexadecimal value surprising; but that is only my own experience, I could be wrong :)

Reply:
> It's just that I always saw netns being referenced as a decimal value (eg. `lsns`) and as a user I find using an hexadecimal value surprising; but that is only my own experience, I could be wrong :)

It's mostly the reason I didn't bother converting it in the first place:

`ls -l /proc/$$/ns/` 

but I changed it here (instead of simply specifying that number can be hex or dec) because seemed a convenient way to show it is also possible to use that (after checking is not uncommon :)

Just to be clear, I'm ok to keep it decimal. I'll simply add a note separately.

Reply:
> Just to be clear, I'm ok to keep it decimal. I'll simply add a note separately.

Looks good to me, I think the doc explicitly mentioning hex is good enough. But if you want, we can add another example with a more natural hex value. One that comes to mind would be `skb->hash` for example.

This made me realize we probably would like to support the `&` operator, as `skb->dev->features & 0xf00` would have been a good hex example. That is of course unrelated to this (and you probably already have it on your todo list) :)

Reply:
That's certainly a nice improvement and should be fairly straightforward to add.

Reply:
in the todo list there's something [involving '&'](https://github.com/retis-org/retis/issues/317) but not exactly fitting. I'll add this.

Reply:
> > Just to be clear, I'm ok to keep it decimal. I'll simply add a note separately.
> 
> Looks good to me, I think the doc explicitly mentioning hex is good enough. 
> [...].
> 

I simply rephrased it to make it clearer.